newIDE/app/src/ObjectGroupsList/index.js

```diff
@@ -499,4 +603,19 @@ export default class GroupsListContainer extends React.Component<Props, State> {
       </Background>
     );
   }
-}
+);
+
+const areEqual = (prevProps: Props, nextProps: Props): boolean =>
```

Comment:
Is this function rightly named? It feels like it's doing the opposite, should this be `areNotEqual` ?

Reply:
Or maybe `shouldUpdate` is better?

Reply:
But the logic of the function actually checks when items are different, so it's doing the opposite no?

Reply:
Oh yes sorry! I was looking at the ObjectsList